kvmagent/kvmagent/test/shareblock_testsuite/test_sblk_metadata.py

```
@@ -0,0 +1,1465 @@
+
+class TestSblkMetadataApi(TestCase, SharedBlockPluginTestStub):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    def _connect_vg(self):
+        iscsi_server = env.get_vm_metadata('self')
+        rsp = storage_device_utils.iscsi_login(iscsi_server.ip, "3260")
+        self.assertEqual(rsp.success, True, "iscsiadm login failed")
+        r, o = bash.bash_ro(
```

Ruff (0.15.7)

[warning] Unpacked variable `r` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)